### DISCUSSION TITLE
Number of Runs VS iter_max in optimization

### DISCUSSION THREAD

**Discussion by leenr01** [gama-platform/gama#73] [Q&A] [answered]
In Prey-Predator model 13, when running the optimization experiment, the number of runs is not the same as the number of max iterations specified in the tabu search method. Why is that? What exactly determines the number of runs? And what does the value of iter_max represent?

**Comment by lesquoyb** [comment]
@ptaillandier can you have a look ?

**Comment by ptaillandier** [comment, marked as answer]
In local search algorithms, the number of iterations corresponds to the number of "moves" in the parameter space. For each move, the algorithm will test the whole neighborhood of the current solution, each neighbor corresponding to a particular set of parameters and thus to a run. Thus, there can be several runs per iteration (maximum: 2^(number of parameters)).

**Comment by ptaillandier** [comment]
I updated the documentation
